nuxt.config.js

```diff
@@ -1,158 +1,183 @@
-const webpack = require('webpack')
-const { base_url, node_env, google_analytics } = require('./config')
-const isDevMode = node_env !== 'production' ? true : false
-
+// eslint-disable-next-line nuxt/no-cjs-in-config
+const { base_url, node_env, google_analytics, firebase } = require("./config");
+const isDevMode = node_env !== "production" ? true : false;
+// eslint-disable-next-line nuxt/no-cjs-in-config
 module.exports = {
-  mode: 'universal',
+  telemetry: false,
   /*
-  ** Headers of the page
-  */
+   ** Headers of the page
+   */
   head: {
     htmlAttrs: {
-      lang: 'tk',
+      lang: "tk",
     },
-    title: 'Serpaý – Goşgular Çemeni',
+    title: "Serpaý – Goşgular Çemeni",
     meta: [
-      { charset: 'utf-8' },
-      { 'http-equiv': 'X-UA-Compatible', content: 'IE=edge' },
-      { name: 'viewport', content: 'width=device-width, initial-scale=1' },
-      { hid: 'description', name: 'description', content: 'Türkmen edebiýatyndan goşgular çemeni' }
-    ],
-    link: [
-      { rel: 'icon', type: 'image/x-icon', href: '/favicon.ico' },
-      { rel: 'icon', type: 'image/png', href: '/favicon.png' }
-    ],
-    noscript: [
-      { innerHTML: 'This website requires JavaScript.' }
-    ],
-    script: [
+      { charset: "utf-8" },
+      { "http-equiv": "X-UA-Compatible", content: "IE=edge" },
+      { name: "viewport", content: "width=device-width, initial-scale=1" },
       {
-        src: `https://www.googletagmanager.com/gtag/js?id=${google_analytics}`, async: true
+        hid: "description",
+        name: "description",
+        content: "Türkmen edebiýatyndan goşgular çemeni",
       },
-      {
-        innerHTML: `window.dataLayer = window.dataLayer || [];function gtag(){window.dataLayer.push(arguments);}gtag('js', new Date());gtag('config', '${google_analytics}');`,
-        type: 'text/javascript',
-        defer: true
-      }
     ],
-    __dangerouslyDisableSanitizers: ['script']
+    link: [
+      { rel: "icon", type: "image/x-icon", href: "/favicon.ico" },
+      { rel: "icon", type: "image/png", href: "/favicon.png" },
+    ],
+    noscript: [{ innerHTML: "This website requires JavaScript." }],
+    // script: [
+    //   {
+    //     src: `https://www.googletagmanager.com/gtag/js?id=${google_analytics}`,
+    //     async: true,
+    //   },
+    //   {
+    //     innerHTML: `window.dataLayer = window.dataLayer || [];function gtag(){window.dataLayer.push(arguments);}gtag('js', new Date());gtag('config', '${google_analytics}');`,
+    //     type: "text/javascript",
+    //     defer: true,
+    //   },
+    // ],
```

Comment:
@bakmyradov Why did you comment out the google analytics here?

Reply:
I didn't have the correct credentials and it was giving an error that's why I left it commented.